lib/run-script.js

```diff
@@ -125,9 +125,16 @@ function run (pkg, wd, cmd, args, cb) {
   } else {
     if (!pkg.scripts[cmd]) {
       if (cmd === "test") {
-        pkg.scripts.test = "echo \"Error: no test specified\"";
+        pkg.scripts.test = "echo \"Error: no test specified\""
+      } else if (cmd === "env") {
+        log.verbose("run-script using default multiplatform env")
```

Comment:
on the verbose message or the command alias? i think having the same name for both is a feature for people writing build scripts that actually checks any of those var.